src/applications/proxy-rewrite/tests/teamsite-header-footer.vrt.spec.js

```diff
@@ -0,0 +1,51 @@
+const puppeteer = require('puppeteer');
+
+const SCREEN_SIZES = [
+  ['full screen', { width: 1024, height: 800 }],
```

Comment:
These tables were a really important part of the test setup to me. I think for VRT we should aim for breadth instead of depth- so as many tests with as little configuration as possible. 

The trade off is that we may exclude small things from the tests like the "Last Updated" section from the footer. I think we should avoid trying to test difficult-to-automate app states unless that state applies to a significant portion of the test suite. e.g. this test script hides the foresee pop-up but it doesn't try to open the megamenu

Reply:
Does this mean we're cutting off screenshots at 800px? That seems reasonable for the teamsite tests.